src/rdkafka_admin.c

```diff
@@ -2911,6 +2913,23 @@ const char *rd_kafka_ResourceType_name(rd_kafka_ResourceType_t restype) {
 }
 
 
+int8_t map_from_resource_type_to_config_resource_type(
```

Comment:
Given we're the treating the Java types let's use the camelcase for function name and the reverse function name.
It should return `rd_kafka_ConfigResourceType_t`.
```suggestion
rd_kafka_ConfigResourceType_t rd_kafka_ResourceType_to_ConfigResourceType(
```